lib/database/group.js

```
@@ -158,4 +158,171 @@ async function groupDB(type, options, method) {
+
+  if (['set', 'add', 'delete'].includes(method)) {
+    // Single type expected
+    filter = filter[0];
+    type = type[0];
+  }
+
+  const dbData = await groupDb.findOne({ where: { jid: options.jid } });
+
+  // ---------- SET ----------
+  if (method === 'set') {
+    if (typeof options.content !== filter[type]) return;
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)